src/hashbang.sh

```diff
@@ -156,99 +166,158 @@ echo " over the internet, and thus by nature we won't even know what it is";
 echo " ";
 
 for keytype in id_rsa id_dsa id_ecdsa id_ed25519; do
-    if [ -e ~/.ssh/$keytype.pub  ]; then
-        if ask " We found a public key in [ ~/.ssh/$keytype.pub ]. Use this key?" Y; then
-            keyfile="~/.ssh/$keytype.pub"
-            key=$(cat ~/.ssh/$keytype.pub)
-            break
-        fi
-    fi
+	if [ -e ~/.ssh/$keytype.pub ]; then
+		if ask " We found a public key in [ ~/.ssh/$keytype.pub ]. Use this key?" Y; then
+			keyfile="~/.ssh/$keytype.pub"
+			key=$(cat ~/.ssh/$keytype.pub)
+			break
+		fi
+	fi
 done
 
+makekey() {
```

Comment:
This function should probably be at the top with the other functions, but that might be just me nitpicking.